src/index.js

```diff
@@ -0,0 +1,155 @@
+  return response;
+};
+
+const addLike = async (event) => {
+  const heart = event.target;
+  const response = await createLike(heart.id);
+  if (response === true) {
+    heart.src = redHeart;
+    const container = heart.parentElement;
+    const hidden = container.childNodes[0];
+    const previous = hidden.value;
+    const actual = parseInt(previous, 10) + 1;
+    hidden.value = actual;
+    const label = container.childNodes[2];
+    label.innerText = `${actual} likes`;
+  }
+};
+
+const getCommentsById = async (itemId) => {
+  const comments = await getComments(itemId);
+  const countComments = (comments === undefined) ? 0 : comments.length;
+  return { countComments, comments };
+};
+
+const getReservationsById = async (itemId) => {
+  const reservations = await getReservations(itemId);
+  const countReservations = (reservations === undefined) ? 0 : reservations.length;
+  return { countReservations, reservations };
+};
+const submitComment = async (e) => {
+  e.preventDefault();
+  const inputName = document.getElementById('name');
+  const inputInsights = document.getElementById('insights');
+  const id = document.getElementsByName('mealId')[0].value;
+  addComment(id, inputName.value, inputInsights.value);
+  const commments = document.getElementsByClassName('comments')[0];
+  const liComment = document.createElement('li');
+  const date = new Date();
+  liComment.innerHTML = `${date.getFullYear()}-${date.getMonth()}-${date.getDay()} ${inputName.value}: ${inputInsights.value}`;
+  commments.appendChild(liComment);
+  const form = document.getElementsByTagName('form')[0];
+  form.reset();
+};
+const closePopup = () => {
+  const popupContainer = document.getElementsByClassName('popupContainer')[0];
+  popupContainer.style.display = 'none';
+  document.removeChild(popupContainer);
+};
+const showCommentsInfo = async (event) => {
+  const button = event.target;
+  const mealId = button.id;
+  const mealInfo = await getMealInfo(mealId);
+  if (mealInfo !== undefined) {
+    const meal = Object.values(mealInfo.meals)[0];
+
+    const { countComments, comments } = await getCommentsById(mealId);
+    let commentsSection = '';
+    if (countComments > 0) {
+      comments.forEach((comment) => {
+        commentsSection += `<li>${comment.creation_date} ${comment.username}: ${comment.comment}</li>`;
+      });
+    }
+    const popupContainer = document.createElement('div');
+    popupContainer.classList.add('popupContainer');
+    const popup = document.createElement('div');
+    popup.classList.add('popup');
+    popup.innerHTML = `<div class='heading'> <a class='close' href='#'><img src='${close}' alt='close icon'></a></div>
+  <div class="content"><div ><div><img class="dishPop" src="${meal.strMealThumb}" alt="mealImage"></div><h1>${meal.strMeal}</h1><p class='instructions' >${meal.strInstructions}</p></div>
+  <h2>Comments(${countComments})</h2>
+  <ul class='comments'>${commentsSection}</ul>
+  <form>
```

Comment:
- [ ] **Required:** You have done an amazing design so far. Kudos. 👏 However, there are small changes to look similar to the design template as follows:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/164944203-dd4755a5-a867-4ed5-8c97-2d7b52e0fcf7.png) | ![image](https://user-images.githubusercontent.com/27709832/164944230-bfcbf0fc-d66d-48c1-91a5-a348723cb4bd.png)

- Kindly add some padding to the comments to push it close to the centre of the page as shown in the template.

Reply:
Resolved 👍🏽